### PR TITLE
Bulk delete redesign

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

Updating documentation for bulk delete redesign 

[Screenshots of feature](https://github.com/medic/cht-core/issues/7778#issuecomment-1292997027).

medic/cht-core#7778

[CHT_Core PR](https://github.com/medic/cht-core/pull/7846)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
@esthermmoturi can you please help me with this small PR? Thank you!

@n-orlowski in case you want to take a look :) Thanks!

**Comment by esthermmoturi** [comment]
> @esthermmoturi can you please help me with this small PR? Thank you!
> 
> @n-orlowski in case you want to take a look :) Thanks!

@latin-panda I am on it!

**Review by esthermmoturi — APPROVED** [review]

File: content/en/apps/features/reports/_index.md [hunk 1]
```diff
@@ -52,15 +52,21 @@ Using the search box, you can search for reports by patient name, phone number,
 
 ## Action Buttons
 
-{{< figure src="reports-bulk.png" link="reports-bulk.png" class="right col-6 col-lg-3" >}}
+The action buttons at the bottom of the screen are configurable using [permissions]({{< ref "apps/reference/app-settings/user-permissions" >}}). Options include submit, edit, delete, review and export reports.
 
-The action buttons at the bottom of the screen are configurable. Options include adding or completing a care guide, bulk select & export. 
+Clicking on the “Export” button will download a CSV file with all the data from the reports.
 
-Clicking the “+” button opens a menu of forms a user can choose to complete. “Bulk Select,” represented by a checkmark icon within a circle, allows you to bulk select and delete multiple reports at a time.
```
[the remark this answers is not in the document]
Comment (latin-panda) [reply content/en/apps/features/reports/_index.md:59]: Do you mean to keep this piece `Clicking the “+” button opens a menu of forms a user can choose to complete`?

**Comment by latin-panda** [comment]
Hi @esthermmoturi, I've applied the feedback, can you please have a look to see if it's okay? Thank you!

**Review by esthermmoturi — APPROVED** [review]
@latin-panda This is good!

**Pull request merged**